libminifi/include/utils/ThreadPool.h

```diff
@@ -340,6 +337,10 @@ class ThreadPool {
   std::recursive_mutex manager_mutex_;
   // thread pool name
   std::string name_;
+  // count of running tasks by ID
+  std::unordered_map<TaskId, unsigned> running_task_count_by_id_;
```

Comment:
is there a reason to use `unsigned` here?  types with a well-defined size like `uint32_t` are generally better

Reply:
Done.